dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/ProgramInstanceQueryParams.java

```diff
@@ -129,6 +137,11 @@
      */
     private boolean includeDeleted;
 
+    /**
+     * List or order params
+     */
+    private List<OrderParam> order;
```

Comment:
Clean code usually suggests, in order to reduce verbosity, to add specific names and no comments, if we want to add an attribute that is a **List or order params** (i think it is also mispelled.. list **of** order params?) we should just call it orderParamsList or similar. Same with other attributes like **pageSize** , do we need a comment to state that is used for page size :) ? ... also considering that there is an effort for less code (see @Accessors)

Reply:
yes, comment is not very usefull here, I can remove it.
On the naming, I understand there are better naming, but this is a frontend parameter, so it needs to be "order". Same for pageSize.

Reply:
ok i didn't know about front end parameter... yes in my opinion comments should be removed unless explaining something important, but that is my opinion